### PR TITLE
feat: footer stub

### LINKED ISSUE #35: Move Header and Footer from Page component to routes root layout.
mb, this needs to be done

### PR DESCRIPTION
**Pull request by sohamjaiswal**

### CODE REVIEW

File: client/src/lib/components/fragments/Page/Page.svelte [hunk 1]
```diff
@@ -114,4 +115,5 @@
 	<div class="content">
 		<slot />
 	</div>
+	<Footer />
 </main>
```
Comment (sohamjaiswal) [inline client/src/lib/components/fragments/Page/Page.svelte]: footer ko main se bahar nikal

**Review by sohamjaiswal — COMMENTED** [review]
read comments

File: client/src/lib/components/fragments/Footer/Footer.svelte [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+			opacity: 1;
+			margin-bottom: 0;
+		}
+	}
+</style>
+
+<script lang="ts">
+</script>
+
+<footer>
```
Comment (sohamjaiswal) [inline client/src/lib/components/fragments/Footer/Footer.svelte]: move this up to the page component

**Comment by sohamjaiswal** [comment]
image

just a tiny modification, drop it a little lower ig?

**Comment by sohamjaiswal** [comment]
image


align it to the center

**Comment by sohamjaiswal** [comment]
this was 1080p btw

File: client/src/lib/components/fragments/Page/Page.svelte (hunk 1 above)
Comment (Sahil2004) [reply client/src/lib/components/fragments/Page/Page.svelte]: I think this is done.

File: client/src/lib/components/fragments/Footer/Footer.svelte (hunk 2 above)
Comment (Sahil2004) [reply client/src/lib/components/fragments/Footer/Footer.svelte]: I think this is also done.

**Comment by Sahil2004** [comment]
> image
> 
> just a tiny modification, drop it a little lower ig?

Fixed.

**Comment by sohamjaiswal** [comment]
lgtm 👍

**Pull request merged**